hatchery/helpers.go

```diff
@@ -123,10 +123,21 @@ func MakeARequestWithContext(ctx context.Context, method string, apiEndpoint str
 	return resp, nil
 }
 
+func useInteralSerices() bool{
+	useInternalServicesValue, useInternalServices := os.LookupEnv("GEN3_USE_INTERNAL_SERVICES")
+	if useInternalServices && strings.ToLower(useInternalServicesValue) == "false" {
+		useInternalServices = false
+	}
+	if useInternalServices && useInternalServicesValue == "0" {
+		useInternalServices = false
+	}
```

Comment:
can `GEN3_USE_INTERNAL_SERVICES` be a config setting instead of an environment variable?
This way we can make it a boolean and only check for boolean values here. Also easier to configure

Reply:
I moved my configuration value from the environmental variable into the config variable and updated the docs.